makefiles/tools/serial.inc.mk

```diff
@@ -4,9 +4,9 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 
 # set default port depending on operating system
 ifeq ($(OS),Linux)
-  PORT ?= $(call ensure_value,$(PORT_LINUX),No port set)
```

Comment:
This change was included for a reason, why are you removing it?

Reply:
Because PORT_LINUX and PORT_DARWIN have always a default value now.

Reply:
Ahh missed the above line